_install-and-configure/install-opensearch/docker.md

```diff
@@ -170,6 +170,7 @@ services:
       - cluster.initial_cluster_manager_nodes=opensearch-node1,opensearch-node2 # Nodes eligible to serve as cluster manager
       - bootstrap.memory_lock=true # Disable JVM heap memory swapping
       - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # Set min and max JVM heap sizes to at least 50% of system RAM
+      - OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_INITIAL_ADMIN_PASSWORD}    # Sets the demo admin user password when using demo configuration, required for OpenSearch 2.12 and higher
```

Comment:
This looks good to me, but I think we may want to call out whether this is set here, or whether this env variable is present from outside or an .env file. Tagging @DarshitChanpura @peterzhuamazon to leave their comments here as well.

Reply:
we should add a note on line 157 stating the requirement to either define the variable in a `.env` and exporting it in the same terminal session where docker-compose is run from.

Reply:
Also, I have a companion PR to update the sample downloadable `docker-compose.yml` file: https://github.com/opensearch-project/project-website/pull/2526